src/nvim/macros.h

```diff
@@ -133,6 +133,8 @@
 /// error. A mechanism to detect many (though not all) of those errors at
 /// compile time is implemented. It works by the second division producing
 /// a division by zero in those cases (-Wdiv-by-zero in GCC).
+//
+// -V:ARRAY_SIZE:1063
```

Comment:
- commit message is a good place for the full explanation though
- i think we should still use the `///` (triple slash) form though , otherwise doxygen might not understand that the docstring is associated with `ARRAY_SIZE` ?